src/cpp/library/string

```diff
@@ -899,10 +893,11 @@ public:
     }
     one[k] = '\0';
     string tmp(one);
+    delete[] one; // Freeing dynamically allocated memory
```

Comment:
Can't we replace the dynamic behavior with a VLA?

Reply:
Implemented as suggested.